README.md

```diff
@@ -162,35 +195,102 @@ Once this is done, there are two ways to run the operator:
 
 ### 1. Run as a Deployment inside the cluster
 
-Build the memcached-operator image and push it to a registry:
+Build the memcached-operator image and push it to your registry as the following example using https://quay.io.
+
+**NOTE**: You are able to you the https://hub.docker.com/ if you wish. 
 
 ```sh
-$ operator-sdk build quay.io/example/memcached-operator:v0.0.1
-$ sed -i 's|REPLACE_IMAGE|quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
+$ operator-sdk build quay.io/<user>/memcached-operator:v0.0.1
+$ sed -i 's|REPLACE_IMAGE|quay.io/<user>/memcached-operator:v0.0.1|g' deploy/operator.yaml
 # On OSX use:
-$ sed -i "" 's|REPLACE_IMAGE|quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
-$ docker push quay.io/example/memcached-operator:v0.0.1
+$ sed -i "" 's|REPLACE_IMAGE|quay.io/<user>/memcached-operator:v0.0.1|g' deploy/operator.yaml
+$ docker push quay.io/<user>/memcached-operator:v0.0.1
+```
+
+The above command will replace the string `REPLACE_IMAGE` with the <image>:<tag> build and pushed to your registry then verify that your `operator.yaml` file was updated successfully. Following an example.
+
+```yaml
+serviceAccountName: memcached-operator
+containers:
+- name: memcached-operator
+  # Replace this with the built image name
+  image: quay.io/example/memcached-operator:v0.0.1
+  command:
+  - memcached-operator
+  imagePullPolicy: Always
 ```
 
+**IMPORTANT:** Ensure that your cluster is able to pull the image pushed to your registry.
+
 The Deployment manifest is generated at `deploy/operator.yaml`. Be sure to update the deployment image as shown above since the default is just a placeholder.
 
 Setup RBAC and deploy the memcached-operator:
 
 ```sh
-$ kubectl create -f deploy/service_account.yaml
+$ kubectl create -f deploy/service_account.yaml 
 $ kubectl create -f deploy/role.yaml
 $ kubectl create -f deploy/role_binding.yaml
 $ kubectl create -f deploy/operator.yaml
 ```
 
-Verify that the memcached-operator is up and running:
+**NOTE:** To apply the RBAC you need be logged in as a user with cluster privileges, for example with the system:admin user to test it. (E.g. By using: `oc login -u system:admin.`) 
+
+Verify that the memcached-operator deployment is up and running in the `memcached` namespace :
 
 ```sh
 $ kubectl get deployment
 NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 memcached-operator       1         1         1            1           1m
 ```
 
+Verify that the memcached-operator pod is up and running:
```

Comment:
I would put tick marks around memcached-operator -> `memcached-operator`

Reply:
Goo catcher. Added in all places that it was missing.